app/assets/javascripts/jquery_overrides.js

```diff
@@ -1,6 +1,10 @@
 jQuery.evalLog = function (text) {
   try {
-    jQuery.globalEval(text.slice('throw "error";'.length));
+    if (text.match('^throw "error";')) {
```

Comment:
the javascript that is being brought in via Apache's ErrorDocument 401 /proxy_pages/invalid_sso_credentials.js would not have the throw "error" in it, this code was blindingly slicing that many characters from the javascript causing a syntax error in browser while rendering it.